test/acsets/CSetDataStructures.jl

```diff
@@ -362,9 +358,9 @@ for lset_maker in lset_makers
   # Pretty-printing with unitialized data attribute.
   lset = lset_maker(Symbol)
   add_part!(lset, :X)
-  @test contains(sprint(show, lset), "#undef")
-  @test contains(sprint(show, MIME"text/plain"(), lset), "#undef")
-  @test contains(sprint(show, MIME"text/html"(), lset), "#undef")
+  @test contains(sprint(show, lset), "nothing")
+  @test contains(sprint(show, MIME"text/plain"(), lset), "nothing")
+  @test contains(sprint(show, MIME"text/html"(), lset), "nothing")
```

Comment:
This result is fine for now but once #692 is implemented, this should be `"nothing"` for nothing-able attributes and `"#undef"` otherwise.